### DISCUSSION TITLE
cmp popup hides nvim-autopair inline fast wrap choices

### DISCUSSION THREAD

**Discussion by zaynOm** [LazyVim/LazyVim#3733] [Q&A]
I disabled mini.pairs as it doesn't seam to have fast wrap feature, I used nvim-autopair instead, but I'm facing the use when I trigger fast wrap after typing the name of a function the auto completion popup is still active and I can't see the fast wrap position keys.

https://github.com/LazyVim/LazyVim/assets/63248283/5025de2e-65af-4c26-909d-1c7803cc9ddb

**Comment by xzbdmw** [comment]
If you setup fast wrap keymap with c-e, you can define a keymap in cmp
```lua
                ["<C-e>"] = cmp.mapping(function(fallback)
                    if cmp.visible() then
                        cmp.close()
                    end
                    fallback()
                end),
```

**Reply by zaynOm** [reply]
I did but still the same, here is how I've done it tell me if I'm missing something.

```lua
return {
  {
    "hrsh7th/nvim-cmp",
    opts = function(_, opts)
      local cmp = require("cmp")
      opts.mapping = vim.tbl_extend("force", opts.mapping, {
        ["<M-e>"] = cmp.mapping(function(fallback)
          if cmp.visible() then
            cmp.close()
          end
          fallback()
        end),
      })
    end,
  },
}
```
